### ISSUE TITLE
TypeError: Cannot set property 'startTime' of undefined

### ISSUE THREAD

**Issue by aleksandrlat** [DataDog/browser-sdk#481]
We are using datadog in our app
We started to see this error in our error logs today
```
File webpack:///./node_modules/@datadog/browser-core/esm/xhrProxy.js line 43 col 27 in proxyXhr
this._datadog_xhr.startTime = performance.now();
```

I cannot repro this myself.

But looks like related to this PR https://github.com/DataDog/browser-sdk/pull/468

Impact is not clear.

Could some one please clarify if we can rollback to some older version of browser-sdk?

**Comment by bcaudan** [comment]
Hi @aleksandrlat, thanks for raising that.

It should be fixed by  #484

**Comment by bcaudan** [comment]
Available in `v1.15.1`.
